infrastructure/src/main/java/org/corfudb/infrastructure/configuration/CorfuConfig.java

```diff
@@ -3,18 +3,15 @@
 import lombok.Getter;
 import org.corfudb.infrastructure.ServerContext;
 
-import java.util.Map;
 
 /**
  * Global configuration for Corfu.
  */
 @Getter
 public class CorfuConfig {
```

Comment:
Now this class only has ServerContext, and has no additional functionality, can we just use ServerContext instead of this wrapper class?

Reply:
+1

Reply:
I agree but in the future it should be a separate config class